core/src/main/java/io/aiven/kafka/tieredstorage/config/RemoteStorageManagerConfig.java

```diff
@@ -95,6 +97,11 @@ public class RemoteStorageManagerConfig extends AbstractConfig {
         + "Remote Log Segment metadata on Remote Log Metadata Manager back-end. "
         + "Allowed values: " + Arrays.toString(SegmentCustomMetadataField.names());
 
+    private static final String UPLOAD_RATE_LIMIT_BYTES_CONFIG = "upload.rate.limit.bytes";
```

Comment:
Maybe the name should be `upload.rate.limit.bytes.per.second` ?  Otherwise it might have confusion on whether the limit is upload size

Reply:
From the bucket4j doc, looks like this rate.limit param actually controls the upper bound (or burst rate) of the bucket, the normal steady rate of tokens in the bucket is controlled by chunk.size.  for this reason maybe the param should be named: `upload.burst.rate` ?

Reply:
Good ideas. If we use rate limit for capacity and refill as [suggested here](https://github.com/Aiven-Open/tiered-storage-for-apache-kafka/pull/548#discussion_r1608976970), then we can just improve the config name as suggested initially to `upload.rate.limit.bytes.per.second`